docs/VPN_README.md

```diff
@@ -13,16 +13,62 @@ without help from Continu. To this end we also have a simple [WireGuard](https:/
 # The WireGuard VPN
 
 To access the [WireGuard](https://www.wireguard.com/) VPN users must first [install
-WireGuard](https://www.wireguard.com/install/). Users then use WireGuard to generate a public and a private key. Keys
-can be generated [as per the docs](https://www.wireguard.com/quickstart/#key-generation) though Windows and OSX clients
-also have GUI ways to do this (TODO: users of these platforms please add details and screen shots).
+WireGuard](https://www.wireguard.com/install/). 
+
+Setting up WireGuard for a user consists of the following steps
+
+1. Select a unique IP address for the user
+2. Have the user generate a public/private key pair
+3. Install the user on the server
+4. Have the user create a client configuration
+5. Have the user test their connection to the server
+
+How the user does some of these steps depends on whether they are using the command-line or GUI version of WireGuard. 
+
+## 1. Select a unique IP address for the user
+
+NOTE: Steps 1 and 2 can take place in parallel. 
+
+Figure out what the next free IP address is by looking at [the server config file](../ansible/templates/wg0.conf). Note that each user has a different number for the `AllowedIPs` line between `100.65.0.` and `/32`. Look at the number for the last user in the list, and increment that number to select the user's IP address. Send the user this IP address so they can create their configuration. 
```

Comment:
@colindavey here and below please line-wrap this to 120 characters - the READMEs probably get read via `cat` or `less` on the command line more than they do in a browser so having them line wrapped is really handy.